### PR TITLE
Adds SCP 216

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Adds the fascinating and **cool** safe - SCP 216!
You simply input the code and store items in there.
On Initialization, there will be randomly generated occupied codes with random items in there. Good luck on the treasure hunt!

## Why It's Good For The Game

More cool(?) SCPs that don't kill you.

## Changelog

:cl:
add: Added SCP-216. It is a safe with near infinite storage space. Some slots come pre-occupied with (potentially) treasure!
/:cl:

### CODE REVIEW

File: code/modules/SCP/SCPs/SCP-216.dm [hunk 1]
```diff
@@ -0,0 +1,187 @@
+/obj/structure/scp_216
+	name = "SCP 216"
+	desc = "A metalic safe with munerical code lock."
```
Comment (Tsurupeta) [inline code/modules/SCP/SCPs/SCP-216.dm]: dude

Comment (EgorDinamit) [reply code/modules/SCP/SCPs/SCP-216.dm]: I wasn't paying attention to description.....

Comment (TheDarkElites) [reply code/modules/SCP/SCPs/SCP-216.dm]: "munerical" lmao

File: code/modules/SCP/SCPs/SCP-216.dm [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+		if(code in all_codes)
+			i -= 1
+			continue
+
+		GenerateRandomItemsAt(code)
+
+/obj/structure/scp_216/Destroy()
+	for(var/atom/A in contents) // Forever gone
+		QDEL_NULL(A)
+	return ..()
```
Comment (Tsurupeta) [inline code/modules/SCP/SCPs/SCP-216.dm]: I have doubts that this'll work

**Review by Tsurupeta — CHANGES_REQUESTED** [review]
`Destroy()` proc won't work because of your associations in `all_codes` list

**Review by Tsurupeta — APPROVED** [review]

**Pull request merged**